src/workloads/execution/EventCollection.yml

```diff
@@ -0,0 +1,71 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Execution
+Description: |
+  Inserts many documents into an event collection.
+
+GlobalDefaults:
+  Collection: &Collection Collection0
+  Database: &db test
+
+Actors:
+  - Name: CreateEventCollection
+    Type: RunCommand
+    Phases:
+      - Repeat: 1
+        Database: *db
+        Operation:
+          OperationName: RunCommand
+          OperationCommand:
+            drop: *Collection
```

Comment:
Do other workloads start with a drop like this? I think we probably don't want this -- it should always be running on a system where the collection doesn't yet exist, as far as I know